From d393f899e2168f107149f329195dcf3968ffab40 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Mon, 18 Dec 2023 19:49:44 +0100
Subject: [PATCH 0774/1141] dt-bindings: i2c: stm32: add 'power-domains'
 property

STM32 I2C may be in a power domain which is the case for the STM32MP25
based boards. Allow a single 'power-domains' entry for STM32 I2C.

Fix dtbs_check warning:
Unevaluated properties are not allowed ('power-domains' was unexpected)

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Icf551f9685d5781ab9cdac726775f11ccd859505
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/347862
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 Documentation/devicetree/bindings/i2c/st,stm32-i2c.yaml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/i2c/st,stm32-i2c.yaml b/Documentation/devicetree/bindings/i2c/st,stm32-i2c.yaml
index 482368d55a8b..fd8ae5122566 100644
--- a/Documentation/devicetree/bindings/i2c/st,stm32-i2c.yaml
+++ b/Documentation/devicetree/bindings/i2c/st,stm32-i2c.yaml
@@ -93,6 +93,9 @@ properties:
   clocks:
     maxItems: 1
 
+  power-domains:
+    maxItems: 1
+
   dmas:
     items:
       - description: RX DMA Channel phandle
-- 
2.39.2

